distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -691,8 +691,12 @@ private static Integer findPortNumber(Integer portToTry) {
           installStartupFile(
               getClass().getResource("/etc/forms/imagery.xml"), "/etc/forms/imagery.xml"),
           installStartupFile(
-              getClass().getResource("/etc/forms/contact-name.xml"),
-              "/etc/forms/contact-name.xml"));
+              getClass().getResource("/etc/forms/contact-name.xml"), "/etc/forms/contact-name.xml"),
+          installStartupFile(
```

Comment:
For the record, I don't like creating a policy file that never exists in the production code. Using templated strings to create the file content feels squishy. However, I don't see a good way around it; it's just part of the price to be paid for using pax exam.